.gitmodules

```diff
@@ -1,41 +1,40 @@
 [submodule "lib/nrfx"]
-	path = lib/nrfx
-	url = https://github.com/NordicSemiconductor/nrfx.git
+    path = lib/nrfx
+    url = https://github.com/NordicSemiconductor/nrfx.git
```

Comment:
Can you please undo all the whitespace changes in this file?
Also, some seem to be reordered. Was this intentional?

Reply:
Neither the spaces nor the order were intentional, my IDE did the formatting. I will revert these chnages.